.travis.yml

```diff
@@ -21,7 +21,10 @@ os:
 language: cpp
 sudo: false
 cache: ccache # https://docs.travis-ci.com/user/caching/
+compiler: clang
```

Comment:
This change appears to have no current impact and I worry that it would lead to hidden changes in the future.  Could you remove this?